### ISSUE TITLE
Feature request - log location of some friends

### ISSUE THREAD

**Issue by Mike570057** [rodit/SnapMod#283]
A cool feature would be to be able to save location of selected users when they share their position to a parsable log file.

**Comment by RevealedSoulEven** [comment]
First let him come back

**Comment by Mike570057** [comment]
> First let him come back

Yes of course, but it doesn't change something to write it here when I think about it... :)
